### PR TITLE
Updated Changelog

### PR DESCRIPTION
**Pull request by matigonkas**
As sugested in: https://github.com/SiCKRAGETV/sickrage-issues/issues/1893

### CODE REVIEW

**Comment by fernandog** [comment]
@matigonkas thanks! dates of the news one are wrong.

if you have the time can you do until v4.0.28 ?
just change the values in the URL

https://github.com/SiCKRAGETV/SickRage/compare/v4.0.20...v4.0.21
https://github.com/SiCKRAGETV/SickRage/compare/v4.0.21...v4.0.22
....

**Comment by neoatomic** [comment]
@matigonkas Thanks for helping out. :)

**Comment by matigonkas** [comment]
@fernandog sure i can do all of those ,i just need the exact release dates of each version as i cant find them

**Comment by MGaetan89** [comment]
Thanks @matigonkas  for doing this!

You can find the dates [here](https://github.com/SiCKRAGETV/SickRage/releases).

File: CHANGES.md [hunk 1]
```diff
@@ -1,3 +1,159 @@
...
+
+[full changelog](https://github.com/SiCKRAGETV/SickRage/compare/v4.0.20...v4.0.21)
+
+* Fixed quality colors on history and displayShow pages
+* Fixed manual adding of group to black or white list
+* Added several network logos
+* Added the ability to browse possible links found in the homepage for the EZTV provider
+* Changed the InitScripts to use variable names and file names referencing SickRage, instead of SickBeard, to prevent conflicts, and improve readability
+* Feature: Add Fanart to local image cache 
+### 4.0.20 (2015-05-10)
```
Comment (MGaetan89) [inline CHANGES.md]: Missing a blank line between line 114 and line 115 ;)

**Comment by MGaetan89** [comment]
Great job @matigonkas !

**Comment by matigonkas** [comment]
fixed @MGaetan89 ,thanks for taking a look :)

**Comment by fernandog** [comment]
@matigonkas thanks!!

**Comment by matigonkas** [comment]
np @fernandog. if you are interresed i dont mind doing the changelog for every release (providing that i have the free time ofc :p )

**Comment by fernandog** [comment]
Ok! I would like. Thanks.

**Comment by miigotu** [comment]
@matigonkas It would be nice if you can squash these commits, and I can merge then. Do you know how?

**Pull request merged**